SerializationErrorCatch/src/main/java/cheetah/example/serializationerrorcatch/function/DeserializationSchema.java

```diff
@@ -0,0 +1,33 @@
+package cheetah.example.serializationerrorcatch.function;
```

Comment:
i would put this in a different package. maybe cheetah.example.serializationerrorcatch.(serde/deserialization)